Dockerfile

```diff
@@ -242,6 +243,8 @@ RUN apt-get update && apt-get install -y \
 	libnl-3-200 \
 	--no-install-recommends
 
+RUN locale-gen C.UTF-8
```

Comment:
When using `C.UTF-8`, I don't think generating is necessary, is it? (`locale -a` should already have it without any additional package installation even)

Reply:
hm.. good point; actually didn't check if it was needed (went by a discussion about the error I ran into on yamllint)

Reply:
On my (quite standard) Ubuntu laptop:
```
# locale-gen C.UTF-8
Error: 'C.UTF-8' is not a supported language or locale
# locale -a | grep -w C
C
C.UTF-8
```